cypress/support/page-objects/cart-page.js

```diff
@@ -0,0 +1,41 @@
+class cartPage {
```

Comment:
Take into consideration to use CamelCase

Reply:
Updated, tnx!